### DISCUSSION TITLE
更新以后没有ctrl+space关闭输入法的状态了

### DISCUSSION THREAD

**Discussion by NEUGWB** [rime/home#1678] [General]
十一之前还好好的

**Comment by fxliang** [comment]
0.16.2的变更，可以用下面的命令切换加旧版本的状态
`WeaselSetup.exe /toggleime`

**Reply by YYMMYB** [reply]
现在似乎不管用了?

**Reply by fxliang** [reply]
0.17.4试了依然可用，未知你提的是什么问题

**Reply by YYMMYB** [reply]
抱歉抱歉, 是我搞错了, 我是想取消这个快捷键, 而不是关闭输入法. 主要是因为这个快捷键与vscode的快捷键冲突了.

**Reply by fxliang** [reply]
> 抱歉抱歉, 是我搞错了, 我是想取消这个快捷键, 而不是关闭输入法. 主要是因为这个快捷键与vscode的快捷键冲突了.

高级键盘设置还是什么的里面有设定系统的快捷键

**Reply by lotem** [reply]
那個系統設置經常自動改回啓用 Ctrl+space
